voxel51/users/api.py

```diff
@@ -458,6 +458,30 @@ def delete_data(self, data_id):
         res = self._requests.delete(endpoint, headers=self._header)
         _validate_response(res)
 
+    def batch_update_data_ttl(self, data_ids, days):
```

Comment:
this will need to support `days=None, expiration_date=None` as per https://github.com/voxel51/api-py/pull/64

Reply:
I'm debating whether this branch or #64 should be updated; #64 depends on a currently-unreleased API feature, while this PR does not (i.e. this could be merged sooner, once it's ready to go)

Reply:
good point. feel free to merge this one in first